docsite-astro/public/index.js

```diff
@@ -0,0 +1,4 @@
+import '/js/easing.js';
+import '/js/animations.js';
+import '/js/slider.js';
+import '/js/scrollspy.js';
```

Comment:
move this file (and referenced files) to `src/` so they can be bundled and minified at build time.